Blueprints/BlueprintDefinitions/vs2022/AnnotationsFramework/template/src/BlueprintBaseName.1/serverless.template

```diff
@@ -20,7 +20,7 @@
           "AWSLambdaBasicExecutionRole"
         ],
         "PackageType": "Zip",
-        "Handler": "BlueprintBaseName._1::BlueprintBaseName._1.Functions_Default_Generated::Default",
+        "Handler": "BlueprintBaseName.1::BlueprintBaseName._1.Functions_Default_Generated::Default",
```

Comment:
This was a bug in the annotations template that it was using `BlueprintBaseName._1` instead of `BlueprintBaseName.1`. What this effect would have been is if the assembly name was `My-Assembly` by using `BlueprintBaseName._1` it would get translated incorrectly here to `My_Assembly`. Luckily for Annotations it doesn't really matter because the annotaton's source generator would have corrected the handler string.